docs/extensions/scala.md

```diff
@@ -55,7 +55,11 @@ export interface ScalaTestParams {
    * It is the result of `buildTarget/scalaTestClasses`. */
   testClasses?: ScalaTestClassesItem[];
 
-  /** The jvm options to run tests with. */
+  /** The jvm options to run tests with. In case of 
+   * conflict (some jvm options from a higher level) this (a) and the other (b) options should be 
+   * concatenated. In case the same jvm option appears 
+   * in both (a) an (b) than the one from (a) takes precedence.
```

Comment:
I find the description to be a little confusing, as to what "higher level" is here. Does this capture your meaning?

> The JVM options to run tests with. In case JVM options are also defined on the build server, they should be concatenated with these. In case they conflict, these should take precedence.

Reply:
Yes, well written. Thanks!
Will update the comment.